include/libtorrent/config.hpp

```diff
@@ -116,7 +116,7 @@ POSSIBILITY OF SUCH DAMAGE.
 // on OSX, use the built-in common crypto for built-in
 # if !defined TORRENT_USE_LIBCRYPTO && !defined TORRENT_USE_LIBGCRYPT
 #  define TORRENT_USE_COMMONCRYPTO 1
-# endif // TORRENT_USE_OPENSSL
+# endif
```

Comment:
was this comment incorrect?

Reply:
I think it's incorrect, the if is not with that macro